pkgs/top-level/python-packages.nix

```diff
@@ -1027,7 +1027,7 @@ in {
   nvchecker = callPackage ../development/python-modules/nvchecker { };
 
   numericalunits = callPackage ../development/python-modules/numericalunits { };
-  
```

Comment:
There's a couple of unwanted spaces changes

Reply:
oops!

Reply:
Fixed.